massa-consensus-worker/src/consensus_worker.rs

```diff
@@ -699,7 +698,7 @@ impl ConsensusWorker {
                     {}
                 );
                 if response_tx
-                    .send(self.block_db.get_export_block_status(&block_id))
+                    .send(self.block_db.get_export_block_status(&block_id)?)
```

Comment:
What do you mean? 

The change is wrapping the returned `Option` into a `Result`, and the error case is a missing block(when it should in fact have been stored because it is present as an active block status).